packages/js/src/first-time-configuration/tailwind-components/base/image-select.js

```diff
@@ -79,6 +80,7 @@ export default function ImageSelect( {
 					id={ url ? id + "__replace-image" : id + "__select-image" }
 					className="yst-button yst-button yst-button--secondary yst-mr-2"
 					onClick={ onSelectImageClick }
+					data-hiive-event-name="clicked_select_image"
```

Comment:
I suppose by the difference `clicked_select_image` knows if it is a replace or not 🤔 
But it could be a separate event too, as you could argue the same for remove.

Reply:
Done.